src/System.Windows.Forms/src/System/Windows/Forms/AxHost.AxContainer.cs

```diff
@@ -107,12 +107,12 @@ MemberInfo[] IReflect.GetMembers(BindingFlags bindingAttr)
                 CultureInfo culture,
                 string[] namedParameters)
             {
-                foreach (DictionaryEntry e in _containerCache)
+                foreach (var e in _containerCache)
```

Comment:
It's better to use deconstruct here:
```suggestion
                foreach ((Control good_variable_name, Control good_variable_name2) e in _containerCache)
```

```cs
Dictionary <int, string> d = new();
d[0] = "a";
d[1] = "b";

foreach ((int n, string s) in d)
{
	Console.WriteLine($"{n}: {s}");
}
```